lua/entities/gmod_wire_vehicle.lua

```diff
@@ -64,7 +64,7 @@ end
 function ENT:BuildDupeInfo()
 	local info = BaseClass.BuildDupeInfo(self) or {}
 
-	if (self.Vehicle) and (self.Vehicle:IsValid()) then
+	if self.Vehicle and (self.Vehicle:IsValid()) then
```

Comment:
Removed parentheses on first case but not second

Reply:
Seems to be a glualinter error, i'll fix it and report the bug ig

Reply:
https://github.com/FPtje/GLuaFixer/issues/158

Reply:
Well go ahead and get rid of the parenthesis so its consistent

Reply:
@wrefgtzweve